docs/migration_guide.md

```diff
@@ -33,13 +33,13 @@ When upgrading your existing Habor instance to a newer version, you may need to
     docker pull vmware/harbor-db-migrator:[tag]
     ```
 
-5. Back up database to a directory such as `/path/to/backup`. You need to create the directory if it does not exist.  Also, note that the username and password to access the db are provided via environment variable "DB_USR" and "DB_PWD"
+5. Back up database to a directory such as `/path/to/backup`. You need to create the directory if it does not exist.  Also, note that the username and password to access the db are provided via environment variable "DB_USR" and "DB_PWD". **MEMTION THAT must BACKUP and RESTORE via `vmware/migratorharbor-db-migrator:1.2` when upgrade from harbor-1.2 or older to harbor-1.3 .** Because on harbor 1.3 the DB engine replaced by MariaDB and MySQL Can not start up through db files backed up by MariaDB.
 
     ```
     docker run -ti --rm -e DB_USR=root -e DB_PWD=xxxx -v /data/database:/var/lib/mysql -v /path/to/backup:/harbor-migration/backup vmware/harbor-db-migrator:[tag] backup
     ```
 
-6.  Upgrade database schema and migrate data:
+6.  Upgrade database schema and migrate data. There are some errors on screen when upgrade from harbor 1.2 to harbor 1.3, just ignore them if not influence harbor to running.:
```

Comment:
Better add a **NOTE:** section as item 7 and it would be more helpful to add the brief error message to clarify.